docs/building.md

```diff
@@ -37,37 +36,37 @@ This, for example, will build a production extension: one that talks to the main
 Building for a local `h` server
 -------------------------------
 
-The default settings file used is `chrome-dev.json` but it does not contain a 
-value for the `oauthClientId`. If you are running your own version of `h` 
-simply create an oauth client ID in the admin UI for the browser extension.
-
-http://localhost:5000/admin/oauthclients
-
-**Client OAuth Values**
-
-    Name: Browser Extension
-    Authority: localhost
-    Grant type: authorization_code
-    Redirect URL: chrome-extension://<extension id>
-
-Where `<extension id>` is the ID of the browser extension. You may need to load the extension 
-into chrome the first to view the ID. Skip ahead to _Loading an extension into Chrome_
-below. Next, add the generated ID value to the config file you are using. e.g inside of 
-`chrome-dev.json` add
-
-```
-"oauthClientId": "<32 digit value>"
-```
-
 > Note:
-> It may be convenient to create a new config file called `custom.json` based 
-> on `chrome-dev.json` with your  `oauthClientId` value.
+> It may be convenient to create a new, untracked config file
+> based  on `chrome-dev.json` to maintain your settings
+> (here called `custom.json`):
 >
 >       $ make SETTINGS_FILE=settings/custom.json
 
-For instructions how to set up a client with your `h` server, see 
-https://h-client.readthedocs.io/en/latest/developers/developing/#running-the-client-from-h*
-
+> These instructions assume you have the `h` service running locally already.
+> For instructions how to set up a client with your `h` server, see 
+> https://h-client.readthedocs.io/en/latest/developers/developing/#running-the-client-from-h*
+
+1. [Create an OAuthClient](http://localhost:5000/admin/oauthclients)
+   for the extension to use in your local instance of `h`, using the following values:
+   
+   ```
+   Name: Browser Extension
```

Comment:
```suggestion
   Name: Chrome Extension
```

Or "Chrome Browser Extension". If the user is building extensions for other browsers it will be helpful if the name indicates which one a particular OAuth client is for.